provisioning/miniprov/hare_mp/main.py

```diff
@@ -201,6 +202,97 @@ def test(args):
         exit(-1)
 
 
+def reset(args):
+    try:
+        rc = 0
+        exit(rc)
+    except Exception as error:
+        logging.error('Error during reset (%s)', error)
+        exit(-1)
+
+
+def cleanup(args):
+    try:
+        rc = 0
+        url = args.config[0]
+        validator = Validator(ConfStoreProvider(url))
+        if validator.is_first_node_in_cluster():
+            if is_cleanup_done():
+                logging.info('No data to cleanup')
+                exit(0)
+
+            nodes = get_all_nodes()
+            rc = config_cleanup(url, nodes) & logs_cleanup(url, nodes)
+
+        exit(rc)
+    except Exception as error:
+        logging.error('Error during cleanup (%s)', error)
+        exit(-1)
+
+
+def is_cleanup_done() -> bool:
```

Comment:
Suggestion: `is_cleanup_needed()`. We don't know whether it is DONE. But we know whether it makes sense to do it.